src/service/ranges/ranges_repository.py

```diff
@@ -12,6 +15,143 @@ def __init__(
         self.query_builder = query_builder
         self.response_sql = response_sql
 
+    def __get_first_range_label(self, max_value: float, label: str) -> dict:
```

Comment:
The functions is returning a string
```suggestion
    def __get_first_range_label(self, max_value: float, label: str) -> str:
```